DeepSpeech.py

```diff
@@ -9,30 +9,23 @@
 os.environ['TF_CPP_MIN_LOG_LEVEL'] = sys.argv[log_level_index] if log_level_index > 0 and log_level_index < len(sys.argv) else '3'
 
 import datetime
-import json
-import numpy as np
+import pickle
 import shutil
 import subprocess
-import tempfile
 import tensorflow as tf
 import time
-import importlib
-import cgi
-import pickle
 
-from collections import OrderedDict
-from math import ceil, floor
+from six.moves import zip, range, filter, urllib, BaseHTTPServer
```

Comment:
Python 3's `range` is Python 2's `xrange`. We could use `range` everywhere and pay a (probably negligible) performance price on Python 2, but why bother when we can just import `six.moves.range` instead? We already have the six dependency anyway.